specs/superchain-configuration.md

```diff
@@ -0,0 +1,68 @@
+# Superchain Configuration
+
+The SuperchainConfig contract is used to manage global configuration values for multiple OP Chains within
+a single Superchain network.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Configurable values](#configurable-values)
+- [Configuration data flow](#configuration-data-flow)
+  - [Pausability](#pausability)
+    - [Paused identifiers](#paused-identifiers)
+    - [Scope of pausability](#scope-of-pausability)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Configurable values
+
+Configurability of the Superchain is currently limited to two values:
+
+The `SuperchainConfig` contract manages the following configuration values:
+
+- `PAUSED_SLOT`: A boolean value indicating whether the Superchain is paused.
+- `GUARDIAN_SLOT`: The address of the guardian, which can pause and unpause the system.
+
+## Configuration data flow
+
+All contracts which read from the `SuperchainConfig` contract hold its address as storage values
+in the proxy account, and call directly to it when reading configuration data.
+
+```mermaid
+flowchart TD
+StandardBridge --> SuperchainConfig
+L1ERC721Bridge --> SuperchainConfig
+L1CrossDomainMessenger --> SuperchainConfig
+OptimismPortal --> SuperchainConfig
+```
+
+### Pausability
+
+The Superchain pause feature is a safety mechanism designed to temporarily halt withdrawals from the system in
+an emergency situation. The Guardian role is authorized to pause and unpause the system.
+
+#### Paused identifiers
+
+The Guardian may distributed a set of presigned transactions to trusted partners, so that the `pause()`
```

Comment:
Preference to reword this, also this has a chance to go stale. I think its leaking a bit of operations into specs. Could argue what the right level of detail is, but perhaps we don't go too deep into the operations in the specs

Reply:
I think that makes sense, I've reworded it just to indicate that it can provide 'additional information'.